definitions/media-roku_app/definition.yml

```diff
@@ -0,0 +1,29 @@
+domain: MEDIA
+# Represents all instances of a customer's Roku App monitored with New Relic's Roku agent.
+type: ROKU_APP
+
+synthesis:
+  identifier: appId
+  name: appName
+
+  encodeIdentifierInGUID: false
+
+  conditions:
+    - attribute: deviceGroup
+      value: Roku
+
+  tags:
+    newRelicAgent:
+    deviceGroup:
+    osName:
+
```

Comment:
I had to remove the `compositeMetrics:` section in order to pass validation (it worked locally but not in the PR checker). Is this an expected result of some schema change? Just want to ensure the metrics will indeed be included 😊

Reply:
It's a change on our side. We are removing the need to define that section. The metrics will be picked up automatically.